portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,216 @@
+                    <h2>Book Store</h2>
+                    <p>May 2020 – Jun 2020</p>
+                    <p>
+                        Bookstore website aimed at storing customer information to be retrieved for future purchases in
+                        a database with all product information dynamically generated using PHP and MySQL. Include files
+                        are used for all code that is used more than once (i.e. search/browse menu, ListAuthor function,
+                        header, and footer).
+                    </p>
+                </div>
+            </div>
+        </div><!-- End of project wrapper. -->
+    </section>
+
+    <section id="social">
+
+        <svg class="social-bsplash" viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg">
+            <path fill="var(--green)"
+                d="M34.4,-60.2C46.2,-52.8,58.5,-46.9,69.4,-37.1C80.3,-27.3,89.7,-13.6,90.4,0.4C91,14.4,82.8,28.7,73.8,41.7C64.8,54.7,54.9,66.4,42.5,75.8C30.1,85.3,15,92.5,-0.7,93.7C-16.4,94.9,-32.8,90.1,-44.5,80.2C-56.1,70.3,-63,55.3,-71.3,41.1C-79.6,26.9,-89.3,13.4,-88.4,0.5C-87.6,-12.4,-76,-24.8,-65.5,-35.2C-54.9,-45.5,-45.3,-53.7,-34.6,-61.6C-23.8,-69.6,-11.9,-77.3,-0.3,-76.8C11.3,-76.3,22.6,-67.5,34.4,-60.2Z"
+                transform="translate(100 100)" />
+        </svg>
+
+        <div class="comic">
+            <div class="caption">enjoy this comic created by me!</div>
+            <img src="images/ghost-comic.png" alt="ghost comic">
+        </div>
+
+        <div class="social-blurb">
+            <h1>Connect with me!</h1>
+            <p>I am available on these forms of social media:
+            <ul>
+                <a href="https://www.linkedin.com/in/sammurraytuesta/">
+                    <li>LinkedIn: linkedin.com/in/sammurraytuesta/</li>
+                </a>
+                <a href="https://github.com/sammurraytuesta">
+                    <li>GitHub: sammurraytuesta</li>
+                </a>
+                <a href="https://gitlab.com/sammurraytuesta">
+                    <li>Gitlab: sammurraytuesta</li>
+                </a>
+                <a href="https://www.instagram.com/sammurraytuesta/">
+                    <li>Instagram: @sammurraytuesta</li>
+                </a>
+                <li>Slack: smurraytuesta@gmail.com</li>
+                <li>Email: smurraytuesta@gmail.com</li>
+            </ul>
+            Feel free to send me a follow or ping me on LinkedIn, Slack, or by Email for any future collaborations!
+            Expect a response anywhere from 2-3 days! I am happy to respond to any business inquiries!
+            </p>
+            <p>Thank you for visiting my portfolio page! I hope you enjoyed your stay! Come again soon!</p>
+
+            <div class="signature">Samantha</div>
+        </div>
+
+    </section>
+
+    <footer>&copy;2020 Samantha Murray Tuesta</footer>
+
+    <!-- smooth scroll -->
+    <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
```

Comment:
Be sure to read http://youmightnotneedjquery.com. Like the site says... *you might not need jquery*.   :-)

Reply:
Haha you're right! I didn't know there was a css trick for smooth scrolling! Thank you!